packages/xo-server/src/api/vm.mjs

```diff
@@ -1327,6 +1329,84 @@ importFromEsxi.params = {
   vm: { type: 'string' },
 }
 
+export async function importMultipleFromEsxi({
+  concurrency = 2,
+  host,
+  network,
+  password,
+  sr,
+  sslVerify = true,
+  stopSource = false,
+  stopOnError = false,
+  thin = false,
+  user,
+  vms,
+}) {
+  const task = await this.tasks.create({ name: `importing vms ${vms.join(',')}` })
+  Task.set('total', vms.length)
+  Task.set('done', 0)
+  Task.set('progress', 0)
```

Comment:
Invalid, these calls must be made from inside the task.

Here they apply to a parent task if any, otherwise there are ignored.